routes/auth.js

```diff
@@ -56,6 +132,51 @@ router.post('/login', (req, res) => {
     })
 });
 
+// create token adminshop
+router.post('/login/admin', (req, res) => {
+    const formData = {
+        email: req.body.email,
+        password: req.body.password,
+    };
+
+    connection.query('SELECT * FROM adminshop WHERE email = ?', [formData.email], (err, user) => {
+        if (err) {
+            res.status(500).send(err);
+        } else {
+            const isSame = bcrypt.compare(req.body.password, user[0].password)
```

Comment:
je le vois plus comme ça, est ce que ça te vas?`
```suggestion
            const isSame = bcrypt.compare(formData.password, user[0].password)
```